lib/chef/mixin/powershell_exec.rb

```diff
@@ -20,14 +20,15 @@
 # The powershell_exec mixin provides in-process access to PowerShell engine via
 # a COM interop (installed by the Chef Client installer).
 #
-# powershell_exec returns a Chef::PowerShell object that provides 3 methods:
+# powershell_exec returns a Chef::PowerShell object that provides 4 methods:
```

Comment:
`provides the following methods:`?

or just "amongst our methods are `result`, `errors`, `error?`, `error!`, an almost fanatical devotion to the Pope, and nice red uniforms..."

Reply:
I was just updating the existing comment